libs/features/game-tele/src/select-game-tele/select-game-tele.integration.spec.ts

```diff
@@ -0,0 +1,170 @@
+import { ComponentFixture, TestBed, waitForAsync } from '@angular/core/testing';
+import { Router } from '@angular/router';
+import { MysqlQueryService, SqliteService } from '@keira/shared/db-layer';
+import { SelectPageObject, TranslateTestingModule } from '@keira/shared/test-utils';
+import { ModalModule } from 'ngx-bootstrap/modal';
+import { ToastrModule } from 'ngx-toastr';
+import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
+import { of } from 'rxjs';
+import { SelectGameTeleComponent } from './select-game-tele.component';
+import Spy = jasmine.Spy;
+import { instance, mock } from 'ts-mockito';
+import { GameTeleHandlerService } from '../game-tele-handler.service';
+import { GameTele } from '@keira/shared/acore-world-model';
+
+describe('SelectConditions integration tests', () => {
+  class SelectGameTelePage extends SelectPageObject<SelectGameTeleComponent> {}
+
+  let queryService: MysqlQueryService;
+  let querySpy: Spy;
+  let navigateSpy: Spy;
+  let router: Router;
```

Comment:
same as above, let's have them `const` inside the `setup` function and return them from there please